app/src/main/java/com/saotome/imed_app/ui/fragments/EditDrugFragment.kt

```diff
@@ -35,7 +40,15 @@ class EditDrug : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_edit_drug, container, false)
+        _binding = FragmentEditDrugBinding.inflate(inflater, container, false)
+        return _binding?.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
```

Comment:
Please add a space line between "super.onViewCreated(view, savedInstanceState)" and the click listener code (like you did in the `onViewCreated `method in the `DrugListFragment`, let's be consistent)